cfme/containers/image.py

```diff
@@ -128,25 +182,82 @@ def compliant(self):
+            image_rows: list of rows from Container Images table
+        """
+        cls._assign_or_unassign_policy_profiles(image_rows, True, *policy_profile_names)
+
+    @classmethod
+    def unassign_policy_profiles(cls, image_rows, *policy_profile_names):
+        """ Unssign Policy Profiles to selected container images rows
+
+        Args:
+            policy_profile_names: :py:class:`str` with Policy Profile names. After Control/Explorer
+                coverage goes in, PolicyProfile objects will be also passable.
+            image_rows: list of rows from Container Images table
+        """
+        cls._assign_or_unassign_policy_profiles(image_rows, False, *policy_profile_names)
+
+    @staticmethod
+    def _assign_or_unassign_policy_profiles(image_rows, assign, *policy_profile_names):
+        """DRY function for managing policy profiles.
+
+        See :py:func:`assign_policy_profiles` and :py:func:`assign_policy_profiles`
+
+        Args:
+            image_rows: list of rows from Container Images table
+            assign: Wheter to assign or unassign.
+            policy_profile_names: :py:class:`str` with Policy Profile names.
+        """
+        images_view = navigate_to(Image, 'All')
+        images_view.table.sort_by('Name', 'asc')
+        images_view.check_rows(image_rows)
+        images_view.select_policy('Manage Policies')
+        # get the object of the Manage Policies view
+        manage_policies_view = get_or_create_current_appliance().browser \
+            .create_view(ProvidersManagePoliciesView)
+        for policy_profile in policy_profile_names:
+            item = manage_policies_view.policies.expand_path(policy_profile)
+            logger.info('manage_policies_view.policies.is_checked() == {}'
+                        .format(manage_policies_view.policies.is_checked(item)))
+            manage_policies_view.policies.check_uncheck_node(assign, policy_profile)
+            logger.info('manage_policies_view.policies.is_checked() == {}'
+                        .format(manage_policies_view.policies.is_checked(item)))
+        manage_policies_view.save.click()
+        flash.assert_no_errors()
+
 
 class ImageAllView(ContainerObjectAllBaseView):
     TITLE_TEXT = "Container Images"
     configuration = Dropdown('Configuration')
 
+    def select_configuration(self, desired_conf, handle_alert=True):
+        self.configuration.item_select(desired_conf, handle_alert)
+
+    def check_rows(self, rows):
+        """check rows on Container Images table."""
+        for row in rows:
+            row[0].check()
+            logger.info('Image ID "{}", have been checked.'.format(row.id.text))
+
 
 @navigator.register(Image, 'All')
 class All(CFMENavigateStep):
     VIEW = ImageAllView
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
+    def am_i_here(self):
+        return match_page(summary='Container Images')
```

Comment:
**REQUIRED**
Not needed, please define `is_displayed` property for `ImageAllView`.